databases/db48/files/yosemite-libtool.patch

```diff
@@ -2,12 +2,15 @@ Index: dist/configure
 ===================================================================
 --- dist/configure.orig
 +++ dist/configure
-@@ -8840,7 +8840,7 @@ $as_echo "$lt_cv_ld_exported_symbols_lis
+@@ -8840,9 +8840,9 @@
        case ${MACOSX_DEPLOYMENT_TARGET-10.0},$host in
  	10.0,*86*-darwin8*|10.0,*-darwin[91]*)
  	  _lt_dar_allow_undefined='${wl}-undefined ${wl}dynamic_lookup' ;;
 -	10.[012]*)
 +	10.[012][,.]*)
  	  _lt_dar_allow_undefined='${wl}-flat_namespace ${wl}-undefined ${wl}suppress' ;;
- 	10.*)
+-	10.*)
++	10.*|11.*)
```

Comment:
Should probably allow any initial number >= 10, after all macOS 12 isn't that far away.

Reply:
I should have mentioned that it only causes an error with `db48 +tcl`.

Reply:
I changed the patch to be more in line with [Ticket 61488](https://trac.macports.org/ticket/61488).

Reply:
> I should have mentioned that it only causes an error with `db48 +tcl`.

Ah, good to know. Thanks.